server/src/main/scala/controllers/UsersController.scala

```diff
@@ -51,11 +52,14 @@ class UsersController @Inject() (
     } yield Ok(Json.toJson(response))
   }
 
-  def getLogs() = handleGET { request =>
+  def getLogs(scrollId: String,limit: String) = handleGET { request =>
+      logger.info(s"limit: $limit")
```

Comment:
This log is useless without any context, also, the arguments need to be updated to `scrollId: Option[String], limit: Int`